source/Core/ModuleList.cpp

```diff
@@ -74,11 +74,18 @@ static constexpr PropertyDefinition g_properties[] = {
      "Control the use of external tools or libraries to locate symbol files. "
      "On macOS, Spotlight is used to locate a matching .dSYM bundle based on "
      "the UUID of the executable."},
+    {"use-swift-dwarfimporter", OptionValue::eTypeBoolean, false, true, nullptr,
+     {}, "Reconstruct Clang module dependencies from DWARF "
+         "when debugging Swift code"},
     {"clang-modules-cache-path", OptionValue::eTypeFileSpec, true, 0, nullptr,
      {},
      "The path to the clang modules cache directory (-fmodules-cache-path)."}};
 
-enum { ePropertyEnableExternalLookup, ePropertyClangModulesCachePath };
+enum {
+  ePropertyEnableExternalLookup,
```

Comment:
These enum values are a little opaque. Can you add a comment?

Reply:
Same here, it's a pattern. We don't document property enums where they are declared, the documentation is in the struct above. Eventually we'll redo how properties are implemented and consolidate everything in a less hand-written fashion. See tha (later) comments in https://reviews.llvm.org/D43647.